common/include/model_rsa.h

```diff
@@ -0,0 +1,30 @@
+#include "verilog_int.h"
+#include <iostream>
+#include <systemc>
+
+constexpr int kBW = 256;
+using KeyType = verilog::vuint<kBW>;
```

Comment:
vuint accepts unsigned, while this works in C++, but not a good practive and might not work under languages like rust :)